### DISCUSSION TITLE
Migration to v5 (MapCachingManager)

### DISCUSSION THREAD

**Discussion by mauriceraguseinit** [JaffaKetchup/flutter_map_tile_caching#54] [Q&A] [answered]
How can i handle a code like this in v5?

```
 FutureBuilder(
      future: MapCachingManager.normalCache,
      builder: (context, snapshot) { 
      ...
```
Because I can't find a MapChachingManager class in v5

**Comment by JaffaKetchup** [comment, marked as answer]
Hi, thanks for your interest.

You should check the new documentation website available from the v5 branch's README. This is the best way to get started with the new version.

In terms of `MapCachingManager`: it doesn't exist any more. Check out the Usage instructions on the website.

Happy to answer any more questions or would love to hear any feedback! Out of interest, what are you using this library for, if you can say?

**Reply by JaffaKetchup** [reply]
Also please consider donating if you can, or giving a star and like! I greatly appreciate it.
